loadtest/store/memstore/random.go

```diff
@@ -399,6 +400,18 @@ func (s *MemStore) RandomCategory(teamID string) (model.SidebarCategoryWithChann
 	return category, nil
 }
 
+func (s *MemStore) RandomProperty() *model.PropertyField {
+	s.lock.RLock()
+	defer s.lock.RUnlock()
```

Comment:
This is a bug, because `s.GetCPAFields()` also gets a read lock. Nested mutexes (even if read-only) will create issues.